OasisDEX/oasis-borrow
# 🧬 [Oasis Create Data] Create cache system + Aave V2 handler

# [[Oasis Create Data] Create cache system](https://app.shortcut.com/oazo-apps/story/9098/oasis-create-data-create-cache-system)
# [[Oasis Create Data] Create handler for Aave V2](https://app.shortcut.com/oazo-apps/story/9495/oasis-create-data-create-handler-for-aave-v2)

  
## Changes 👷‍♀️
- enforced `LendingProtocol` usage everywhere
- changes to `LendingProtocolLabel`
- changes to `AssetsTableRowData` regarding the db schema
- pulling db data, validations on `/api/product-hub`
- mocking db data method
- updating data methods
- some basic updating db stuff
- aave v2 handler
  
## How to test 🧪
- enable the feature
- go to `/oasis-create/borrow`
- there should be _nothing_
- start the db again with a clear state
- migration should work OOTB with launch.sh but you can run it manually using `yarn migrate`
- use the mocks to load up the db with mock data
- oasis create page should be populated with data now
